internal/appsec/remoteconfig.go

```diff
@@ -197,7 +197,9 @@ func (a *appsec) handleASMFeatures(u remoteconfig.ProductUpdate) map[string]rc.A
 			log.Error("appsec: Remote config: error while unmarshalling %s: %v. Configuration won't be applied.", path, err)
 		} else if data.ASM.Enabled && !a.started {
 			log.Debug("appsec: Remote config: Starting AppSec")
-			if err = a.start(); err != nil {
+			telemetry := newAppsecTelemetry()
+			defer telemetry.emit()
```

Comment:
what about moving this into `appsec.start()`?

it also took me some time to realize the state was saved in globals ^^'

Reply:
`appsec.start()` is not called if:
- The WAF is unhealthy (platform unsupported, failed to load)
- AppSec is pending remote activation (`DD_APPSEC_ENABLED` unset)
- The configuration is invalid
- ...

This would result in us not getting any information about places where AppSec does not _actually_ run.